### DISCUSSION TITLE
[灌水][委托]几年前的白祁重工社长卷款跑路案是否就那么简单？

### DISCUSSION THREAD

**Discussion by LV0Incapableperson** [share121/inter-knot#4954] [灌水]
各位认为，白祁重工社长卷款跑路事件是否就这么简单？是不是疑点有点太多了？治安局有点结案太潦草了？这是我找来的几年前艾利都沦陷后，新艾利都成立不久时，治安局发的通报，已经被删了。时间有点久，可能比较糊，但是具体可以看得出来，仅仅只是消失失踪就莫名其妙把旗下资金转移的罪魁祸首扣在了
霍尔斯头上，而且在通告里出现了推测两个字，他们到底有去调查吗？治安局是真的去调查了，还是调查不出个所以然所以草草结案？15年发的通报，16年11月就结案了，是不是因为拖了太久没有任何进展？治安局到底负责任了吗？到现在在网上还是没有什么关于霍尔斯的消息，这个社长到底哪去了？没有人想过吗 ？治安局没有想过去找白祁重工社长吗？为什么他们第一时间不是去找人而是直接扣帽子？这就是治安局的作风跟嘴脸吗？这个社会还真是险恶啊，治安局还值得我们信任吗？我相信这位社长一定是清白的，或许是掉入了空洞什么的，因为他活不见人死不见尸，进入空洞迷失是唯一的可能了。有没有绳将愿意接委托去空洞里尝试寻找一下？委托金再说。 

![image](https://github.com/user-attachments/assets/a2b9a412-3036-441f-916b-1c5eec802c45)

**Comment by WinterSamu** [comment]
让子弹多飞一会吧。

**Comment by sb35386057** [comment]
2015年？9年了？就交代这些？吃干饭的？中介我很生气😡请求彻查

**Reply by KELERSAMA** [reply]
不 他们调查了这件事一年多就草草结案了

**Comment by yyyzmo** [comment]
哥们，照片糊成shi了
